### ISSUE TITLE
[BUG]: App closes abruptly after swiping pages in the reader

### ISSUE THREAD

**Issue by Dev-Tan** [avluis/Hentoid#1193]
**Hentoid Version**: v1.19.12

**Device/Android Version**: Samsung Galaxy S10+ / Android 12

**Issue details / Repro steps**: 
* Actual behavior: When I try to read a book, the app abruptly closes itself and reboots. This can be repeated infinitely. The exact moment the app closes can't be exactly pinpointed. It can happen after 10-page swipes, but also after just one. I didn't test if it only happens to archives or also with already extracted folders.

* Expected behavior: The app won't crash, no matter how many books I read. 

**Stack trace / LogCat**:
```ruby
latest-crash log : begin
Hentoid ver: 1.19.12 (744)
API: 31
Device: SM-G975F
Start : 2024-10-15T01:38:31.853Z
End : 2024-10-15T01:38:31.853Z (00:00)
-----

java.util.ConcurrentModificationException
	at java.util.ArrayList$Itr.next(ArrayList.java:860)
	at me.devsaki.hentoid.viewmodels.ReaderViewModel.clearForceReload(Unknown Source:19)
	at me.devsaki.hentoid.fragments.reader.ReaderPagerFragment.differEndCallback(Unknown Source:69)
	at me.devsaki.hentoid.fragments.reader.ReaderPagerFragment.onImagesChanged$lambda$40$lambda$39(Unknown Source:5)
	at me.devsaki.hentoid.fragments.reader.ReaderPagerFragment.$r8$lambda$giHgo132JVD6CqaR3Q7DTeyOl1A(SourceFile:0)
	at me.devsaki.hentoid.fragments.reader.ReaderPagerFragment$$ExternalSyntheticLambda8.run(SourceFile:0)
	at androidx.recyclerview.widget.AsyncListDiffer.onCurrentListChanged(Unknown Source:26)
	at androidx.recyclerview.widget.AsyncListDiffer.latchList(Unknown Source:15)
	at androidx.recyclerview.widget.AsyncListDiffer$1$2.run(Unknown Source:16)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:226)
	at android.os.Looper.loop(Looper.java:313)
	at android.app.ActivityThread.main(ActivityThread.java:8751)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:571)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1135)

latest-crash log : end
```

@AVnetWS/admin-team

**Comment by RobbWatershed** [comment]
Thanks for the detailed feedback. This has been fixed for next release

**Comment by RobbWatershed** [comment]
Fixed on today's v1.20.0
